CHANGELOG.md

```diff
@@ -11,6 +11,46 @@ Refer to [changes](./changes) directory for unreleased changes.
 
 <!-- towncrier release notes start -->
 
+# [5.5.0](https://github.com/omnivector-solutions/jobbergate/releases/tag/5.5.0) - 2025-02-18
```

Comment:
Please, revert the changes on this file. We just need it at release time, not on a feature branch.

Reply:
Notice we also need the news fragments back

Reply:
@fschuch you may need to make the requisite changes and push this PR forward.

Reply:
Addressed by [4adadff](https://github.com/omnivector-solutions/jobbergate/pull/732/commits/4adadff00887bbf32d8a8e59e98b64ace8f399e8).